app/src/main/res/values/strings.xml

```diff
@@ -18,6 +18,7 @@
     <string name="downloading">Downloading</string>
 
     <string name="menu_item_none">None</string>
+    <string name="playback_info">Transcoding: %b\n\nVideo streams:\n%s\n\nAudio streams:\n%s</string>
```

Comment:
I think this will become a mess when something gets added and all translations need to be updated

Reply:
> I think this will become a mess when something gets added and all translations need to be updated

Hm, this is true. I'll split them up into multiple strings, and upload some screenshots afterwards.